js/usercss.js

```diff
@@ -0,0 +1,183 @@
+/* globals loadScript mozParser */
+
+'use strict';
+
+function wildcard2regexp(text) {
```

Comment:
We'll get rid of all global functions someday, please don't add any at least in the background page. Move it into usercss object.

Reply:
I'll wrap them into IIFE.

Reply:
Also this function should be removed since we don't use `@include` rule anymore.